ui/raidboss/data/06-ew/raid/p2n.ts

```diff
@@ -0,0 +1,206 @@
+          call: 'getCombatants',
+        });
+        if (!callData || !callData.combatants || !callData.combatants.length) {
+          console.error('SpokenCataract: failed to get combatants: ${JSON.stringify(callData)}');
+          return;
+        }
+        // This is the real hippo, according to hp.
+        const hippos = callData.combatants.filter((c) => c.BNpcID === 13721);
+        if (hippos.length !== 1) {
+          console.error('SpokenCataract: There is not exactly one Hippo?!?: ${JSON.stringify(hippos)}');
+          data.bodyActor = undefined;
+          return;
+        }
+        data.bodyActor = hippos[0];
+      },
+      alertText: (data, matches, _output) => {
+        if (!data.bodyActor) {
+          console.error('SpokenCataract: No boss actor found. Did the promise fail?');
+          return;
+        }
+        const heading = data.bodyActor?.Heading;
+        let bodyHeading = null;
+
+        if (heading < -3.0 || heading > 3.0)
+          bodyHeading = 'north';
+        if (heading < -1.56 && heading > -1.58)
+          bodyHeading = 'west';
+        if (heading > -0.1 && heading < 0.1)
+          bodyHeading = 'south';
+        if (heading > 1.56 && heading < 1.58)
+          bodyHeading = 'east';
+
+        if (matches.id === '67F8') {
+          if (bodyHeading === 'north')
+            return 'body north, head south';
+          if (bodyHeading === 'west')
+            return 'body west, head east';
+          if (bodyHeading === 'south')
+            return 'body south, head north';
+          if (bodyHeading === 'east')
+            return 'body east, head west';
+        }
+        if (matches.id === '67F7') {
+          if (bodyHeading === 'north')
+            return 'body north, head east';
+          if (bodyHeading === 'west')
+            return 'body west, head north';
+          if (bodyHeading === 'south')
+            return 'body south, head west';
+          if (bodyHeading === 'east')
+            return 'body east, head south';
+        }
+        if (matches.id === '67F9') {
+          if (bodyHeading === 'north')
+            return 'body north, head west';
+          if (bodyHeading === 'west')
+            return 'body west, head south';
+          if (bodyHeading === 'south')
+            return 'body south, head east';
+          if (bodyHeading === 'east')
+            return 'body east, head north';
+        }
+      },
+      outputStrings: {
+        text: {
+          en: 'Tank Laser on YOU',
+          de: 'Tank Laser auf DIR',
+          fr: 'Tank laser sur VOUS',
+          ja: '自分にタンクレーザー',
+          cn: '坦克射线点名',
+          ko: '탱 레이저 대상자',
+        },
+      },
+    },
+    {
+      id: 'P2N Sewage Deluge',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67F6', source: 'Hippokampos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Aoe get on grid',
```

Comment:
```suggestion
          en: 'Aoe--Get on grid',
```

Minor style nit, feel free to ignore.

Reply:
Fixed in 2fe82019d5146ac7edffbd41251b7b6aaca4113d.